topbeat/beat/config.go

```diff
@@ -4,9 +4,10 @@ type TopConfig struct {
 	Period *int64
 	Procs  *[]string
 	Stats  struct {
-		System     *bool
-		Proc       *bool
-		Filesystem *bool
+		System       *bool
+		Proc         *bool
+		Filesystem   *bool
+		Cpu_per_core *bool
```

Comment:
Yes, I agree with you it's better to add "yaml". I didn't reach the point to make nicer the code.